fork-common/build.gradle

```diff
@@ -17,26 +17,27 @@ sourceCompatibility = rootProject.sourceCompatibility
 targetCompatibility = rootProject.sourceCompatibility
 
 dependencies {
-    compile project(':fork-client')
-    compile(
-            "com.google.code.findbugs:jsr305:3.0.1",
+    api project(':fork-client')
+    api project(':fork-stat-common')
+    api "com.google.code.findbugs:jsr305:3.0.1"
+    api "com.android.tools.ddms:ddmlib:25.1.0"
+    api "commons-io:commons-io:2.5"
+    api "org.apache.commons:commons-lang3:3.4"
+    api "com.github.spullara.mustache.java:compiler:0.8.0"
+    api "com.google.code.gson:gson:$GSON_VERSION"
```

Comment:
why mixing defined versions with hardcoded?

Reply:
@cdsap good question, I'll move all dependencies description to dependencies.gradle